packages/clay-icon/src/index.tsx

```diff
@@ -43,6 +43,7 @@ const ClayIcon = React.forwardRef<SVGSVGElement, IProps>(
 					`lexicon-icon lexicon-icon-${symbol}`,
 					className
 				)}
+				key={symbol}
```

Comment:
Should we also add a useEffect so that the component doesnt re-render unless symbol or one of the other props changes?

Reply:
I don't think `useEffect` is really needed or desirable here.

Reply:
Hmm, not sure about `useEffect`... why would we use it?

Reply:
oh i mis-read your original post. I thought you were adding this to avoid re-renders, but i see now that its for not re-using the node.